service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -259,6 +275,14 @@ struct TSInsertionReq {
     5: required i64 stmtId
 }
 
+struct TSBatchInsertionReq {
+    1: required string deviceId
+    2: required list<string> measurements
+    3: required list<TSDataValueList> columns
+    4: required list<i64> timestamps
+    5: required list<IoTDBDataType> types
```

Comment:
Is `types ` duplicated with `TSDataValueList.type`?

Reply:
if removed, TSDataValueList can not be used alone. I have removed it, we can add back if needed.